functions/src/create-contract.ts

```diff
@@ -113,6 +116,7 @@ export const createmarket = newEndpoint(['POST'], async (req, auth) => {
     ante,
     closeTime.getTime(),
     tags ?? [],
+    autoResolutionTime,
```

Comment:
This is not in the right order, `autoResolutionTime` should be the last parameter. (To make this more robust, we could switch to taking all the parameters as an object instead of a param list. Not sure if that should be part of this change.)